### PR TITLE
chdb: run clickbench with v1.4.1 chdb

### PR DESCRIPTION
**Pull request by chhetripradeep**
Updated result with latest chdb using:
- MergeTree Engine
- Using chdb's dbapi with persistence.

Will add parquet one in followup PR. 

**Thank You for Your Contribution!**

We appreciate your effort and contribution to the project. To ensure that your Pull Request (PR) adheres to our guidelines, please ensure to review the rules mentioned in our contribution guidelines:

[ClickHouse/ClickBench Contribution Rules](https://github.com/ClickHouse/ClickBench#rules-and-contribution)

Thank you for your attention to these details and for helping us maintain the quality and integrity of the project.

### CODE REVIEW

File: chdb/results/c6a.4xlarge.json [hunk 1]
```diff
@@ -8,48 +8,48 @@
     "load_time": 0,
```
Comment (auxten) [inline chdb/results/c6a.4xlarge.json:8]: In this case, load time should be the insert time

**Comment by auxten** [comment]
Maybe we can rename the old chdb dir to chdb-parquet?

**Comment by chhetripradeep** [comment]
Added back the previous chdb results to chdb-parquet.

**Review by auxten — APPROVED** [review]

**Comment by auxten** [comment]
LGTM

**Pull request merged**